### PR TITLE
add compression-level description

### PR DESCRIPTION
**Pull request by Leguan16**
This adds the necessary description for the new `compression-level` config option in `config/paper-global.yml`

See https://github.com/PaperMC/Paper/pull/9711

### CODE REVIEW

**Comment by Nacioszeczek** [comment]
Waiting for the changes to be merged into Paper

**Review by olijeffers0n — CHANGES_REQUESTED** [review]
Blocking for paper merge. Also, can you doc what the default value is please? In the description that is please ☺️

**Comment by Leguan16** [comment]
well, i checked if default values in the other places (where IntOr is used) are present and they aren't so I did not include it here as well for consistency.

**Comment by lynxplay** [comment]
Not that -1 would be of much use to the docs. The default is up to the deflate impl

**Comment by Leguan16** [comment]
> Not that -1 would be of much use to the docs. The default is up to the deflate impl

does that mean that including the value is unnecessary or not?

**Comment by lynxplay** [comment]
Can include the -1 being default, just you need to also explain that -1 means the compression will choose a sensible compression level.

**Comment by Leguan16** [comment]
> Can include the -1 being default, just you need to also explain that -1 means the compression will choose a sensible compression level.

Should I also mention that Paper is using the velocity compressor and that the default is 6?

**Comment by olijeffers0n** [comment]
Personally i would say yes, as it gives people something to use as a reference to their values as well.

**Comment by Leguan16** [comment]
I hope this makes sense lol

**Comment by olijeffers0n** [comment]
Yeah looks ok 👌

File: docs/paper/admin/reference/configuration/global-configuration.mdx [hunk 1]
```diff
@@ -218,6 +218,12 @@ misc:
     chat-executor-max-size:
       default: '-1'
       description: 'The maximum number of threads to allow in the chat thread pool. The default value is -1, which means no maximum'
+  compression-level:
+    default: 'default'
+    description: 'A higher compression level means less data transmitted for the cost of more CPU time.
```
Comment (powercasgamer) [inline docs/paper/admin/reference/configuration/global-configuration.mdx:223]:
```suggestion
    description: 'A higher compression level means less data transmitted at the cost of more CPU time.
```

**Comment by Leguan16** [comment]
changed

**Review by powercasgamer — APPROVED** [review]

**Review by lynxplay — APPROVED** [review]

**Pull request merged**